package.json

```diff
@@ -96,7 +97,9 @@
           "organizations-storage.phone-numbers.item.get",
           "organizations-storage.urls.collection.get",
           "organizations-storage.urls.item.get",
-          "ui-organizations.third-party-services"
+          "ui-organizations.third-party-services",
+          "organizations-storage.settings.collection.get",
```

Comment:
I see the use of these permissions only in settings, I think in this set they are redundant. Or maybe I missed the place?